### PR TITLE
Add UL/LI in console-section-body to better read

### PR DESCRIPTION
**Pull request by castorls**
Here's a pull request which add a better outline view (with UL/LI).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,16 +28,16 @@ THE SOFTWARE.
...
 
     <artifactId>collapsing-console-sections</artifactId>
     <version>1.6.1-SNAPSHOT</version>
     <packaging>hpi</packaging>
     
     <properties>
-        <jenkins.version>1.609.3</jenkins.version>
-        <java.level>6</java.level>
+        <jenkins.version>2.46</jenkins.version>
+        <java.level>8</java.level>
```
Comment (oleg-nenashev) [inline pom.xml]: It's not going to work in such way.
Firstly, there is n reason to update the LTS dependency. Then, 2.46.x requires compatibility with Java 7.
Please revert these changes or explain why do you need them

**Comment by castorls** [comment]
Here's comme 2 screenshots (before/after) 

before 

image

after 

image

File: pom.xml [hunk 2]
```diff
@@ -37,7 +37,7 @@ THE SOFTWARE.
     
     <properties>
         <jenkins.version>1.609.3</jenkins.version>
-        <java.level>6</java.level>
+        <java.level>7</java.level>
```
Comment (oleg-nenashev) [inline pom.xml]: 1.609.3 is the last LTS baseline compatible with Java 6

**Comment by oleg-nenashev** [comment]
Sorry, forgot about it :(

**Comment by oleg-nenashev** [comment]
Released in 1.7.0

**Comment by forpix** [comment]
Can any body help me, how can I use this plugin to hide the checkout scm step output in Jenkins console output for pipeline jobs(scripted pipeline)
my Jenkinsfile is looks like this
node{
 def scmVars = checkout scm
        checkout scm
        echo 'scm : the commit id is ' +scmVars.GIT_COMMIT
        echo 'scm : the commit branch  is ' +scmVars.GIT_BRANCH
       
} 
**which gives the following output :** 

Cloning the remote Git repository
Cloning repository https://github.com/forpi/cherry-pik.git
 > git init /home/ubuntu/.jenkins/workspace/Dummy-project # timeout=10
Fetching upstream changes from https://github.com/forpi/cherry-pik.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/forpi/cherry-pik.git +refs/heads/*:refs/remotes/origin/*
 > git config remote.origin.url https://github.com/forpi/cherry-pik.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/forpi/cherry-pik.git # timeout=10
Fetching upstream changes from https://github.com/forpi/cherry-pik.git
 > git fetch --tags --progress https://github.com/forpi/cherry-pik.git +refs/heads/*:refs/remotes/origin/*

 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision b80c4d6b655429d7f84347b4192461cc3d68283e (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f b80c4d6b69c429d7f84347b4192461cc3d68283e
 > git branch -a -v --no-abbrev # timeout=10
 > git checkout -b master b80c4d6b655429d7f84347b4192461cc3d68283e
Commit message: "lets try this again" 
 the commit id is :b80c4d6b655429d7f84347b4192461cc3d68283e
 the commit branch  is :refs/remotes/origin/master

I want only commit id and branch without this checkout scm step output ...any leads here

**Comment by oleg-nenashev** [comment]
There is a user mailing list for questions
